pkg/gc/gc_state_manager_test.go

```diff
@@ -849,6 +876,247 @@ func (s *gcStateManagerTestSuite) TestGCBarriers() {
 	re.Empty(res.BlockerDescription)
 }
 
+func (s *gcStateManagerTestSuite) TestGlobalGCBarriers() {
+	re := s.Require()
+
+	now := time.Date(2025, 03, 06, 11, 50, 30, 0, time.Local)
+	re.Empty(s.getGlobalGCBarriers())
+
+	keyspaceID := s.keyspacePresets.manageable[0]
```

Comment:
`manageable[0]` is always the null keyspace. And this doesn't match our test pattern in this file.

For most of the cases in the test where different states in different keyspaces is not relavant, consider replace your calls to `AdvanceTxnSafePoint` to
```go
for _, id := range s.keyspacePresets.manageable {
    s.manager.AdvanceTxnSafePoint(...)
}
```